### PR TITLE
Use new mutation createOfferFillTransaction

### PR DESCRIPTION
**Pull request by NicolasMahe**
### Project organization

- Related https://github.com/liteflow-labs/backend/issues/2404
- Dependant on https://github.com/liteflow-labs/backend/pull/2405

### Description

- use `createOfferFillTransaction` instead of `offer.fill` in `useAcceptOffer` hook

### Checklist

- [x] Update related changelogs

### CODE REVIEW

File: packages/hooks/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
```
Comment (NicolasMahe) [inline packages/hooks/CHANGELOG.md]: @NicolasMahe backend is already released with this feature

File: packages/hooks/src/useAcceptOffer.ts [hunk 2]
```diff
@@ -13,7 +13,6 @@ gql`
   query OfferWithApprovalAndFill(
```
Comment (antho1404) [inline packages/hooks/src/useAcceptOffer.ts]:
```suggestion
  query OfferWithApproval(
```

**Review by antho1404 — APPROVED** [review]
Other than the renaming, the rest is good to go

File: packages/hooks/src/useAcceptOffer.ts (hunk 2 above)
Comment (NicolasMahe) [reply packages/hooks/src/useAcceptOffer.ts]: done

**Pull request merged**